docs/lib/auth/fragments/native_common/authentication/signin/common.md

```diff
@@ -1,6 +1,41 @@
 
-<inline-fragment platform="ios" src="~/lib/auth/fragments/native_common/authentication/register_user/common.md"></inline-fragment>
-<inline-fragment platform="android" src="~/lib/auth/fragments/native_common/authentication/register_user/common.md"></inline-fragment>
+Auth category can be used to register a user into the auth plugin and sign-in the registered user using username and password in your app. AWS Cognito Auth Plugin is set up with AWS Cognito User Pools that manages the users and their properties.
```

Comment:
Hmm. This graf reads oddly. What are we trying to express here? Let me give it a shot:

> The Authentication category can be used to manage your user directory using Amazon Cognito User Pools. Your users can register and authenticate against a User Pool which will store things like their password, email, and other properties.

Be careful with service names -- it's Amazon Cognito, Not AWS Cognito.